### PR TITLE
Added some snippet descriptions to sponges

### PR DESCRIPTION
**Pull request by Drafting-Goblin**
Added 8 snippet descriptions to sponges.

#### Summary

SUMMARY: Content "Added 8 snippet descriptions to sponges"

#### Purpose of change
I noticed that sponged didn't have snippet descriptions, but they seemed like an item that could have them. There are a lot of different types of sponges in the world.

#### Describe the solution

It adds 8 snippet descriptions.

#### Describe alternatives you've considered

To add more flavor to sponges there could be different sponge objects, but as they would all be basically the same this feels like the best approach.

#### Testing

I spawned the sponges and observed the descriptions. I looked in houses to find naturally spawning sponges as well.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
Odd place to add details but I'm cool with it

File: data/json/items/tool/toileteries.json [hunk 1]
```diff
@@ -150,7 +150,17 @@
     "symbol": "s",
     "color": "yellow",
     "use_action": [ "WASH_HARD_ITEMS" ],
-    "flags": [ "NO_SALVAGE" ]
+    "flags": [ "NO_SALVAGE" ],
+    "snippet_category": [
+      { "id": "spng1", "text": "A round smiley face sponge." },
+      { "id": "spng2", "text": "A rectangular sponge with a green abrasive back." },
+      { "id": "spng3", "text": "A natural sea sponge, It looks kind of lumpy." },
```
Comment (I-am-Erk) [inline data/json/items/tool/toileteries.json:157]:
```suggestion
      { "id": "spng3", "text": "A natural sea sponge.  It looks kind of lumpy." },
```

File: data/json/items/tool/toileteries.json [hunk 2]
```diff
@@ -150,7 +150,17 @@
...
+      { "id": "spng4", "text": "A dolphin shaped sponge, It looks at home in the water." },
```
Comment (I-am-Erk) [inline data/json/items/tool/toileteries.json:158]:
```suggestion
      { "id": "spng4", "text": "A dolphin shaped sponge.  It looks at home in the water." },
```

**Review by I-am-Erk — CHANGES_REQUESTED** [review]

**Comment by Drafting-Goblin** [comment]
Thanks, I like adding these kinds of details

**Pull request merged**